src/AmbientSounds.Uwp/Services/SoundDataProvider.cs

```diff
@@ -146,14 +146,14 @@ public async Task<bool> IsSoundInstalledAsync(string id)
         /// <inheritdoc/>
         public async Task RefreshLocalSoundsMetaDataAsync()
         {
-            if (_localSoundCache is null || _localSoundCache.Count == 0)
+            if (_localSoundCache is not { Count: > 0 })
```

Comment:
Or is there a performance improvement by using this approach?

Reply:
There is no performance improvement (in fact the two variants produce exactly the same codegen, [see here](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAZgAJiAmcgYQFgAoAbyfPfIAcoBLAN2wYY5AMoQArgDsAJrWxgAFsID6AGwhhsqsVNnylAbjYdj7MuR6SM5ALIBGABQBKU+VaMOniwDNyDtRpaOjJyisI8uOSS4qqq5ISE5AGa2hIh+jAAdLRp1gC8eeQADC4eXuzu5eXEAOzkcHZGZeUAvq6e7Ry15ChUTZ5tzeSu5pbWNlTOrpVVPL7+6inBemEWkZIQ1sx0uSDkAHzF5C2lVW6d1XUN/VWDZxeUdb037IODTObUomkrSkwzZgoYx2Ui25AA5jAMAZyLgoTC3kA===)), it was more about using patterns to merge the two checks (since `is not { Count: > 0 }` handles both the `Count` check as well as the `null` check, in a declarative way), to possibly make the code easier to read at a glance. If you prefer the older syntax though I can revert this though, sure!